TASK-2/CurrencyExchanger/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿
+CurrencyExchanger.CurrencyExchanger.ExchangeCurrency();
```

Comment:
You must declare the Program class and his runner(Main), which will already have this logic.
Do not use features from new versions, like 6.0, when it is just released, and in the work you may encounter old versions.